datastore/prometheus/views.py

```diff
@@ -85,13 +99,74 @@ def _total_num_sources_in_last_run(self):
             NUM_PROBLEM_SOURCES_IN_LAST_RUN.set(-1)
             NUM_OK_SOURCES_IN_LAST_RUN.set(0)
 
+    def _num_current_grants_with_beneficiary_location_geocode_without_lookup(self):
+        current_grants = db.Latest.objects.get(series=db.Latest.CURRENT).grant_set
+
+        # Thought: Shouldn't we check that there are the same number of lookup outputs as there are input geocodes?
+        #  e.g. if a grant has two beneficiary locations w/ geocodes, they should both be represented in the output locationLookup
+        # ... but lookup can also come from recipient org HQ location can't it? (but not for the case of grants to individuals)
+        # TODO: investigate
+
+        # Workaround, fixed in Django 5.0
+        # See: https://stackoverflow.com/questions/42332304/django-an-equality-check-in-annotate-clause
+        class GreaterThan(Func):
+            arg_joiner = ">"
+            arity = 2
+            function = ""
+
+        zero = Cast(0, output_field=IntegerField())
+
+        # This query is a bit gnarley using postgres-specific JSON functions
```

Comment:
No need to change this now but if it becomes more complex than its worth (readability, maintainability) there is prior precedent for falling back to using [raw sql](https://github.com/ThreeSixtyGiving/datastore/blob/master/datastore/data_quality/quality_data.py#L258) , or [partial](https://github.com/ThreeSixtyGiving/datastore/blob/master/datastore/data_quality/quality_data.py#L236)  [etc](https://github.com/ThreeSixtyGiving/datastore/blob/master/datastore/db/management/commands/manage_entities_data.py#L77).